### ISSUE TITLE
Move Collection to its own Nav

### ISSUE THREAD

**Issue by aerni** [statamic/ideas#259]
I'm working on an addon that has a Collection of Products. I would like to move this Products Collection to its own dedicated Nav and remove it from the regular Collections Nav and Collections Listing.

I was able to remove it from the Collections Nav already. But couldn't find a way to remove it from the Collections Listing as well.

```php
Nav::extend(function ($nav) {
  $nav->content('Collections')
    ->children(function () {
      return Collection::all()->sortBy->title()->filter(function ($item) {
        return $item->handle() !== 'products';
      })->map(function ($collection) {
        return Nav::item($collection->title())
          ->url($collection->showUrl())
          ->can('view', $collection);
      });
    });
});

Nav::extend(function ($nav) {
  $nav->create('Products')
    ->section('Snipcart')
    ->route('collections.show', 'products')
    ->icon('drawer-file');
});
```

**Comment by riasvdv** [comment]
https://github.com/riasvdv/statamic-collection-groups 😊

**Comment by jesseleite** [comment]
There's a pretty good chance we'll be implementing pinned nav items.  See the discussion in https://github.com/statamic/ideas/issues/74

**Comment by aerni** [comment]
That's not quite the same, is it? Pinned is great to use for certain collections for sure. But in this case, I'd like to create a whole new Section for an addon and add a collection there. Plus other custom menu items.

**Comment by jesseleite** [comment]
Yeah, your mileage is going to vary, as `Nav::extend()` isn't smart enough to deal with listing pages (and I'm not sure it should).

If you need to _remove_ a collection from the `Collections` nav and listing, you might be able to extend `CollectionRepository` and bind your implementation to the container (see https://statamic.dev/extending/repositories).  Could get messy though, especially within an addon, where your end user (or another addon) might be intentionally binding their own `CollectionRepository` extension.

If there's a valid enough use case, it might be better to add some kind of `hide: true` or `listable: false` setting to a collection's yaml file, to hide the collection from nav and listings automatically.  Would have to think through implications across the control panel though (ie. permissions, etc).

**Comment by aerni** [comment]
Yeah, something like `listable: false` could make sense.

**Comment by mons0630** [comment]
Sorry for popping up the old post, but any one have the solution?

**Comment by jackabox** [comment]
Hey, did you ever find a way to solve this? Wanting to remove a collection myself from the nav/collection overview page but failing.

**Comment by aerni** [comment]
No, there's currently no solution for this.

**Comment by kevinmeijer97** [comment]
Hi! I wanted to follow up on this topic.

In [PR #80](https://github.com/rapidez/statamic/pull/80), we're building runway resources to link Magento categories and products. We aim to enhance these resources by adding custom fields without modifying the Magento database tables. To achieve this, we linked a collection to each runway resource so that the resource displays the fields from the collection.

The only issue we’re facing now is that we’d like to hide these collections from users, as they’re editing the data directly on the runway resource and don’t need to see the collections.

Since [PR #3325](https://github.com/statamic/cms/pull/3325) was closed a few years ago, would it be possible to reconsider adding this feature?
